osu.Game/Skinning/LegacySkin.cs

```diff
@@ -71,6 +73,18 @@ protected override void Dispose(bool isDisposing)
                 case GlobalSkinColour colour:
                     return SkinUtils.As<TValue>(getCustomColour(colour.ToString()));
 
+                case LegacySkinConfigurations legacy:
+                    switch (legacy)
+                    {
+                        case LegacySkinConfigurations.Version:
+                            if (Configuration.LegacyVersion.HasValue)
+                                return SkinUtils.As<TValue>(new Bindable<decimal>(Configuration.LegacyVersion.Value));
```

Comment:
Note: I'm preparing a pull request to ban `HasValue` and `Value` in framework side. Consider to use pattern matching because it has slightly better performance, and good readability.

Reply:
Well I meant 'is decimal version'... If you are not familiar with this I'll cover this in later PR by finding all usages.